src/groups/unitary.jl

```diff
@@ -1,32 +1,35 @@
 @doc raw"""
-     Unitary{n,𝔽} = GeneralUnitaryMultiplicationGroup{n,ℂ,AbsoluteDeterminantOneMatrices}
+     Unitary{n,𝔽} = GeneralUnitaryMultiplicationGroup{n,𝔽,AbsoluteDeterminantOneMatrices}
 
-The group of unitary matrices ``\mathrm{U}(n)``.
+The group of unitary matrices ``\mathrm{U}(n, 𝔽)``, either complex (when 𝔽=ℂ) or quaternionic
+(when 𝔽=ℍ)
 
-The group consists of all points ``p ∈ \mathbb C^{n × n}`` where ``p^\mathrm{H}p = pp^\mathrm{H} = I``.
+The group consists of all points ``p ∈ 𝔽^{n × n}`` where ``p^\mathrm{H}p = pp^\mathrm{H} = I``.
 
 The tangent spaces are if the form
 
 ```math
-T_p\mathrm{U}(x) = \bigl\{ X \in \mathbb C^{n×n} \big| X = pY \text{ where } Y = -Y^{\mathrm{H}} \bigr\}
+T_p\mathrm{U}(n) = \bigl\{ X \in 𝔽^{n×n} \big| X = pY \text{ where } Y = -Y^{\mathrm{H}} \bigr\}
 ```
 
 and we represent tangent vectors by just storing the [`SkewHermitianMatrices`](@ref) ``Y``,
-or in other words we represent the tangent spaces employing the Lie algebra ``\mathfrak{u}(n)``.
+or in other words we represent the tangent spaces employing the Lie algebra ``\mathfrak{u}(n, 𝔽)``.
+
+Quaternionic unitary group is isomorphic to the compact symplectic group of the same dimension.
 
 # Constructor
 
-    Unitary(n)
+    Unitary(n, 𝔽::AbstractNumbers=ℂ)
```

Comment:
is positional the normal way we do? I think so but just to check – For Euclidean we have a keyword, because the number of parameters is not fixed but otherwise it is a positional?

Reply:
Also, `Sphere` has positional and `ArraySphere` has keyword. I thin the reasoning is exactly what you wrote.